routers/users.py

```
@@ -0,0 +1,193 @@
+        detail="자격 증명을 확인 할 수 없습니다.",
+        headers={"WWW-Authenticate": "Bearer"},
+    )
+    payload = decode_access_token(token)
+    if not payload:
+        raise credentials_exception
+    user_id: str = payload.get("sub")
+
+    users = get_db_users()
+    user = next((u for u in users if u["user_id"] == user_id), None)
+
+    token, expires_in = create_access_token(user["user_id"])
```

Ruff (0.14.11)

Unpacked variable `expires_in` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)